### PR TITLE
[Chore] Mhaura/Selbina ships cleanup

### PR DESCRIPTION
**Pull request by Xaver-DaRed**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Cleans "Ship bound to Selbina/Mhaura" (and its pirate counterpart) zone NPCs.
Takes some logic from PR #2168 that fixes "Time left" messages.

## Steps to test these changes

Not much, really.

### CODE REVIEW

File: scripts/zones/Ship_bound_for_Mhaura/npcs/Sahn.lua [hunk 1]
```diff
@@ -12,33 +12,16 @@ entity.onTrade = function(player, npc, trade)
...
-    elseif (vHour ==  2) then
-        vHour = 4
-    elseif (vHour ==  3) then
-        vHour = 3
-    elseif (vHour ==  4) then
-        vHour = 2
-    elseif (vHour ==  5) then
-        vHour = 1
-    end
+    vhour = vhour - (vhour - 3) * 2
```
Comment (TeoTwawki) [inline scripts/zones/Ship_bound_for_Mhaura/npcs/Sahn.lua]: `vHour` not `vhour`

File: scripts/zones/Ship_bound_for_Mhaura_Pirates/npcs/Sahn.lua [hunk 2]
```diff
@@ -12,31 +12,14 @@ entity.onTrade = function(player, npc, trade)
...
-    elseif vHour ==  2 then
-        vHour = 4
-    elseif vHour ==  3 then
-        vHour = 3
-    elseif vHour ==  4 then
-        vHour = 2
-    elseif vHour ==  5 then
-        vHour = 1
-    end
+    vhour = vhour - (vhour - 3) * 2
```
Comment (TeoTwawki) [inline scripts/zones/Ship_bound_for_Mhaura_Pirates/npcs/Sahn.lua]: same here

File: scripts/zones/Ship_bound_for_Selbina/npcs/Bhagirath.lua [hunk 3]
```diff
@@ -12,33 +12,16 @@ entity.onTrade = function(player, npc, trade)
...
-    elseif (vHour ==  2) then
-        vHour = 4
-    elseif (vHour ==  3) then
-        vHour = 3
-    elseif (vHour ==  4) then
-        vHour = 2
-    elseif (vHour ==  5) then
-        vHour = 1
-    end
+    vhour = vhour - (vhour - 3) * 2
```
Comment (TeoTwawki) [inline scripts/zones/Ship_bound_for_Selbina/npcs/Bhagirath.lua]: and here

File: scripts/zones/Ship_bound_for_Selbina_Pirates/npcs/Bhagirath.lua [hunk 4]
```diff
@@ -12,31 +12,14 @@ entity.onTrade = function(player, npc, trade)
...
-    elseif vHour ==  2 then
-        vHour = 4
-    elseif vHour ==  3 then
-        vHour = 3
-    elseif vHour ==  4 then
-        vHour = 2
-    elseif vHour ==  5 then
-        vHour = 1
-    end
+    vhour = vhour - (vhour - 3) * 2
```
Comment (TeoTwawki) [inline scripts/zones/Ship_bound_for_Selbina_Pirates/npcs/Bhagirath.lua]: boy the copy paste got you on all of them today

Comment (Xaver-DaRed) [reply scripts/zones/Ship_bound_for_Selbina_Pirates/npcs/Bhagirath.lua]: it did lol

**Review by claywar — APPROVED** [review]
Seems reasonable after fixing the `vhour` var that Teo and CI called out

**Review by TeoTwawki — APPROVED** [review]

**Pull request merged**